src/server/worldserver/worldserver.conf.dist

```diff
@@ -3622,6 +3622,14 @@ LFG.KickPreventionTimer = 900
 
 EnablePlayerSettings = 0
 
+#
+#     Allow.Join.BG.And.LFG
+#        Description: Allow queueing for BG and LFG at the same time.
+#        Default:     0 - Disabled
+#                     1 - Enabled
+
+Allow.Join.BG.And.LFG = 0
```

Comment:
`JoinBGAndLFG.Enable = 0`

Reply:
@IntelligentQuantum could you check this one please?